### DISCUSSION TITLE
All deployments and redeployments failing

### DISCUSSION THREAD

**Discussion by asyazwan** [vercel/community#187] [Help] [answered]
Since a few hours ago all new builds are failing, and rebuilds of prior successful builds are also failing. Builds are triggered through Github actions and rebuilds manually triggered from within Vercel UI.

The whole thing seems to completely build, but they will fail at the last step:

```
Created all serverless functions in: 2:34.930 (m:ss.mmm)
Uploading build outputs...
Deploying build outputs...
An unexpected internal error occurred
```

Any ideas?

**Comment by leerob** [comment]
1. Are you still seeing this? If so, can you add a link to your project or code?
1. Does your application build locally?

**Comment by asyazwan** [comment, marked as answer]
Hi Lee,

1. Yes, however upgrading to Next 12 seems to fix it. We’re not ready to upgrade yet so we will have to go through everything and make sure our app works with Next 12. Repo is private but project is https://vercel.com/yaqeen/yaqeen-web <https://vercel.com/yaqeen/yaqeen-web>
2. Yes

**Comment by GZLiew** [comment]
Having same issues currently where all deployments are failing 
![image](https://user-images.githubusercontent.com/43195293/142793103-55e828f5-4fe9-48f9-9f7e-d8fdf365d435.png)

My application builds locally just fine

**Reply by GZLiew** [reply]
link 
https://vercel.com/tapendium-team/uat-pwa/47eEoZVtikuxGdZgpTihdM2EKFj8

**Reply by asyazwan** [reply]
Our application was using Next10. Upgrading to Next12 fixes this. :(
